From feb370817f3b99c48419781dd30cdd2d118f4ebe Mon Sep 17 00:00:00 2001
From: Dmitry Shilnenkov <dmitry.shilnenkov@gmail.com>
Date: Sun, 1 May 2022 19:36:09 +0300
Subject: [PATCH] Get rid of compile commands output

This option is also used to perform clang-tidy check of genesis project,
so we must disable it for glslang.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ada2b8fb..69c70ac6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,7 +43,7 @@ endif()
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 # Enable compile commands database
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+# set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 # Adhere to GNU filesystem layout conventions
 include(GNUInstallDirs)
-- 
2.32.0 (Apple Git-132)

